ui/k-toolbar/main.kytos

```diff
@@ -1,20 +1,92 @@
 <template>
   <k-toolbar-item icon="compass" tooltip="Napp Pathfinder">
-    <k-accordion>
-      <k-accordion-item title="Best Path">
-        <k-dropdown icon="circle-o" title="Source:" :options="get_interfaces"
+    <div style="overflow-y: auto; height:550px;">
```

Comment:
It is not recommend to use the inline style attribute for HTML elements.
It would be better to move all style attributes in this file to the style tag.

Reply:
Fixed. Thank you!